CMakeLists.txt

```diff
@@ -730,6 +730,30 @@ if (USE_TORCH)
   set(TORCH_INC_DIR ${TORCH_LOCATION}/include/ ${TORCH_LOCATION}/include/torch/csrc/api/include/ ${CMAKE_BINARY_DIR}/pytorch/src/pytorch/torch/include/torch/csrc/api/include ${TORCH_LOCATION}/.. ${CMAKE_BINARY_DIR}/src)
   set(TORCH_LIB_DIR ${TORCH_LOCATION}/lib/)
 
+  # TORCH VISION
+  message(STATUS "Configuring Pytorch Vision")
+  set(PYTORCH_VISION_COMPLETE ${CMAKE_BINARY_DIR}/CMakeFiles/pytorch_vision-complete)
+  # set(PYTORCH_VISION_COMMIT b68adcf9a9280aef02fc08daed170d74d0892361) # 0.6 & torch 1.5
+  set(PYTORCH_VISION_COMMIT 78ed10cc51067f1a6bac9352831ef37a3f842784) # 0.7 & torch 1.6
+  set(TORCHVISION_LOCATION "${CMAKE_BINARY_DIR}/pytorch_vision/src/pytorch_vision-install")
+  ExternalProject_Add(
+    pytorch_vision
+    PREFIX pytorch_vision
+    INSTALL_DIR ${CMAKE_BINARY_DIR}
+    GIT_REPOSITORY https://github.com/pytorch/vision.git
+    GIT_TAG ${PYTORCH_VISION_COMMIT}
+    GIT_CONFIG advice.detachedHead=false
```

Comment:
I have no idea why we need that, but I think cmake doesn't like it to detect if it should rebuild the project or not.

Reply:
Some patches are applied to pytorch, that's why this kind of warning is removed I think.
On torchvision there are no patches, so it does not matter

Reply:
After research this is a warning remover